rules-tests/CodeQuality/Rector/Class_/InlineConstructorDefaultToPropertyRector/Fixture/skip_early_return.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector\Fixture;
+
+final class SkipEarlyReturn
```

Comment:
as partial changed, please rename class name along with file name to something like "DonotChangeHasBeforeIf" -> "do_not_change_has_before_if.php.inc"

Reply:
Done